infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/AWSProvider.java

```diff
@@ -739,9 +739,10 @@ private boolean doRelease(Script script, Properties inputs, TestPlan testPlan)
                             "SERVER=$(awk -F= '/^NAME/{print $2}' /etc/os-release)\n" +
                             "if [[ $SERVER == '\"Ubuntu\"' ]]; then\n" +
                             "update-rc.d testgrid-agent defaults\n" +
-                            "elif [[ $SERVER = 'CentOS Linux' ]]; then\n" +
+                            "elif [[ $SERVER = 'CentOS Linux' ]] || [[ $SERVER = \"SLES\" ]]; then\n" +
```

Comment:
Shouldn't this be `|| [[ $SERVER = '\"SLES\"' ]]`, can you please check?

Reply:
With single quotes, it won't escape the double-quotes. I've tested and this works.